Exposing the deployed Grafana in the Loki tutorial (#219)
kyma-project/examples: loki/README.md

     ```
     
     Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
  
  ### Expose Grafana
  1. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
     ```bash
     curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
     ```
  1. Create an APIRule:
     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
     ```
  1. Get the public URL of your Loki instance:
     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get vs -l apirule.gateway.kyma-project.io/v1beta1=grafana.${KYMA_LOKI_EXAMPLE_NS} -ojsonpath='{.items[*].spec.hosts[*]}'
     ```

  ### Add a Link for Grafana to the Kyma Dashboard
  1. Download the `dashboard-configmap.yaml` file and change `{GRAFANA_LINK}` to the text you retrieved in the previous step.
     ```bash
       curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/dashboard-configmap.yaml -o dashboard-configmap.yaml
     ```

>      NOTE: You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.

Reviewer: ```suggestion
     >NOTE: You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.
```

     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
     ```
     
     Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
  

>   3. To expose Grafana using the Kyma API Gateway, download the api rule file and replace the `{release-name}` to the name of Helm release:

Reviewer 2: ```suggestion
  3. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
```

     ```bash
     curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
     ```

>      Create an APIRule:

Reviewer 2: It's an instruction, why not numbered?

```suggestion
4. Create an APIRule:
```

     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
     ```

>      Get the public URL of your Kiali server:

Reviewer 2: ```suggestion
5. Get the public URL of your Kiali server:
```

     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME}-grafana 3000:80
     ```
     
     Open Grafana in your browser under `http://localhost:3000` and log in with user admin and the password taken from the previous Helm command.
  
  3. To expose Grafana using the Kyma API Gateway, download the APIRule file and replace the `{release-name}` variable with the name of Helm release:
     ```bash
     curl https://raw.githubusercontent.com/kyma-project/examples/main/loki/apirule.yaml -o apirule.yaml
     ```
4. Create an APIRule:
     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} apply -f apirule.yaml 
     ```

> 5. Get the public URL of your Kiali server:

Reviewer 3: ```suggestion
5. Get the public URL of your Loki instance:
```

     ```bash
     kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get vs -l apirule.gateway.kyma-project.io/v1beta1=grafana.${KYMA_LOKI_EXAMPLE_NS} -ojsonpath='{.items[*].spec.hosts[*]}'
     ```

>   4. To add the link to exposed Grafana to kyma dashboard, we should edit the `dashboard-configmap.yaml` and change the `{grafana-link}` to the text you retrieved from the previous step. You can change `label` field if you wish to change the name of the tab, but also the `category` tab if you wish to move it to another category. After, apply this `ConfigMap`, go to the busola dashboard, and you will see your newely exposed Grafana under Observability section. If you already have some `busola-config`, you should merge it with existing one:

Reviewer: This point 4 looks like another set of instructions, maybe we can use an ordered list, like in previous points? For example:
###Add the link of the exposed Grafana to Kyma Dashboard
1. Edit `dashboard-configmap.yaml` and change `{GRAFANA_LINK}` to the text you retrieved in the previous step.

>**NOTE:** You can change the label field to change the name of the tab, and the category tab if you wish to move it to another category.

3. Apply this ConfigMap, and go to Kyma Dashboard. You should see your newly exposed Grafana under the Observability section.
4. If you already have `busola-config`, merge it with the existing one:
